geopandas/geoseries.py

```diff
@@ -1190,12 +1189,18 @@ def estimate_utm_crs(self, datum_name: str = "WGS 84"):
         """
         return self.values.estimate_utm_crs(datum_name)
 
-    def to_json(self, **kwargs) -> str:
+    def to_json(self, show_bbox=True, drop_id=False, to_wgs84=False, **kwargs) -> str:
         """
         Returns a GeoJSON string representation of the GeoSeries.
 
         Parameters
```

Comment:
`to_wgs84` is missing from the docstring.

Reply:
apologies, have fixed that